Bottom: 4e194d7ca0c059c45fbf42f24afd6fbbe65b3a57
Top:    8bf80d34fb6911975c1ecd5dab97554f8c7ef785
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-01-23 18:51:15 +0000

Hack around Circuit CAM / EDG brokeness reading drill files







---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index a016217..ddfa8f2 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -462,10 +462,17 @@ gerber_set_layer (const char *name, int group)
 	      int ap = findApertureCode (pending_drills[i].diam, ROUND);
 	      fprintf (f, "T%02d\015\012", ap);
 	    }
+#ifdef EDG_ARE_BROKEN
+	  fprintf (f, "X%06ldY%06ld\015\012",
+		   100 * gerberX (PCB, pending_drills[i].x), 100 * gerberY (PCB,
+								pending_drills
+								[i].y));
+#else
 	  fprintf (f, "X%06ldY%06ld\015\012",
 		   gerberX (PCB, pending_drills[i].x), gerberY (PCB,
 								pending_drills
 								[i].y));
+#endif
 	}
       free (pending_drills);
       n_pending_drills = max_pending_drills = 0;
